deployment/live/gcp/example-gcp/ci/terragrunt.hcl

```diff
@@ -0,0 +1,16 @@
+terraform {
+  source = "${get_repo_root()}/deployment/modules/gcp//example-gcp"
```

Comment:
This double slash is a bit weird?

Reply:
This is how you tell terraform about module "boundaries", with the double slash it brings in anything under there, without them it fails when you try to reference the sibling `gcs` from within the `example-gc` module.

This is a bit of a dusty corner in terraform, but it's why e.g. you had to put double slashes in for your distributor config which referenced other modules:https://github.com/transparency-dev/distributor/blob/main/deployment/modules/distributor/main.tf#L110